local/README.md

```diff
@@ -44,6 +39,6 @@ val scoreFn = new OpWorkflowRunnerLocal(workflow).score(opParams)
 Below is an example of measured scoring performance on 6m records with 10 fields and 12 transformations applied.
 Executed on MacBook Pro i7 3.5Ghz in a single thread.
 ```
-Scored 6,000,000 records in 239s
-Average time per record: 0.0399215ms
+Scored 6,000,000 records in 710s
```

Comment:
interesting that there is 3x throughput difference

Reply:
I had to warm up the JVM. Updated the numbers.

Reply:
It's slightly faster.